content/applications/general/product_images.rst

```diff
@@ -0,0 +1,133 @@
+==============
+Product images
+==============
+
+The product images are very useful in Odoo, for example, to quickly find a product or check if you 
+scanned the right one, but it can be a bit painful to set up especially if you have a lot of 
+products. **Google Custom Search** allows finding images automatically for your product, based on 
```

Comment:
"The Product Image module" allows... ? Most other doc pages are bad examples.

@jcs-odoo needs your advice here too.

Reply:
About the **page title**, I don't see any reason to hide Google since that's exactly what this doc is about.
I would go for something like "Automatic product images with Google Image". What's interesting here is the fact that you can populate plenty of product images **automatically**.

"Product images" alone as a page title is borderline misleading.

Reply:
About the description, I personally wouldn't go for what's "painful" nor advertise a feature as a starter. That's a bit too "marketing" for a documentation page ^^ (same for "keeping your focus on what matters in your business").

The documentation isn't a place to sell our product, but to inform.

If you consider my comment above, the automatic population of product images is what's this doc is really about. So the intro would be mostly describing this, using Google Custom Search (in bold, that's fine)